packages/sanity/src/core/search/common/types.ts

```diff
@@ -70,6 +70,7 @@ export interface SearchFactoryOptions {
   /* only return unique documents (e.g. not both draft and published) */
   unique?: boolean
   strategy?: SearchStrategy
+  perspective?: string
```

Comment:
we might want to use this for something in the future, hence why I didn't want to tie it specifically to the "raw" aspect.

Reply:
Good call!